### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_TextTab__Keyboard :"Toggle More " menu item is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5719]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
**Environment Details**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Detail:**
Tool Name: Keyboard
MAS Reference: MAS2.1.1

**Steps to Reproduce :**
1.Install the app Azure Data Studio.
2.Connect to the Database->Go to file->New Notebook.
3.Select query->Click on the Run command->click on text tab
4.Try to navigate “Toggle More ” menu item using keyboard.

**Actual:**
“Toggle More ” menu item is not accessible through keyboard.

**Similar issue is observed for the below pages:** 
                                                                                                                                                                                                                                                                                                    **1.NotebookQueryTable:** Navigate to the connected server->Select the database->Navigate to the  file->New Notebook->Select any query->Click on the Run command

**Expected:**
“Toggle More ” menu item should be accessible through keyboard.

**User Impact:**
User with disabilities find it difficult to navigate to “Toggle More” item list.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:**
**MAS2.1.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference:**
[5719_[Accessibility]A11y_AzureDataStudio_TextTab__Keyboard .pptx](https://github.com/microsoft/azuredatastudio/files/3232437/5719_.Accessibility.A11y_AzureDataStudio_TextTab__Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
@AccessibilityTestingTeam-TCS, this is possible to do when the "Tab Moves Focus" option is enabled in Azure Data Studio.

From the notebook, the keyboard shortcut should be ctrl+shift+m. When you press this combo, you will see the following at the bottom right of the screen:
![image](https://user-images.githubusercontent.com/40371649/58839759-f8ea8680-8617-11e9-99a7-c3d2d23e8998.png)



After that point, go back into the notebook and run through the scenario again. I was able to navigate through it and complete the scenario using the keyboard with this setting enabled:
![tab moves focus](https://user-images.githubusercontent.com/40371649/58839881-6c8c9380-8618-11e9-830d-2fd308cc94be.gif)

**Comment by AccessibilityTestingTeam-TCS** [comment]
I tried the steps you told ,but still it's not working.
[5719_[Accessibility]A11y_AzureDataStudio_TextTab__Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3285459/5719_.Accessibility.A11y_AzureDataStudio_TextTab__Keyboard.pptx)

**Comment by AccessibilityTestingTeam-TCS** [comment]
I checked this issue on following environment 
Version: 1.9.0-insider (user setup)
Commit: ad805b993c95e65e600921145c4f8d7602205ab7
Date: 2019-07-11T07:43:05.264Z
VS Code 1.34.0
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
and the issue is still reproduced.

**Comment by AccessibilityTestingTeam-TCS** [comment]
I checked this issue on following environment
Version: 1.10.0-insider (user setup)
Commit: f720ec642f6ffd51d8772f663a611292f91a563d
Date: 2019-07-15T07:47:15.691Z
VS Code 1.34.0
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
and the issue is still reproduced.
[5719_[Accessibility]A11y_AzureDataStudio_TextTab_Keyboard_Reopened .pptx](https://github.com/microsoft/azuredatastudio/files/3404909/5719_.Accessibility.A11y_AzureDataStudio_TextTab_Keyboard_Reopened.pptx)

**Comment by chlafreniere** [comment]
This can be worked around with the use of Tab Moves Focus.
